eng/PublishSourceBuild.props

```diff
@@ -149,7 +149,7 @@
 
     <!-- Tests expect this file to be in ArtifactsAssetsDir. -->
     <Copy SourceFiles="$(SourceBuiltTarballName)"
-          DestinationFolder="$(ArtifactsAssetsDir)"
+          DestinationFolder="$(ArtifactsAssetsDir)source-build"
```

Comment:
A property should be defined for this path so it can be shared both here and in the `Publishing.props` file.

Reply:
Update: https://github.com/dotnet/dotnet/pull/1827/commits/36fdcff8a83adc3d875d2357d270639638a2dc56